components/TripsListElement/TripsListElement.tsx

```
@@ -0,0 +1,20 @@
+
+
+const trips: TripData[] = [
+  {title: "wycieczka poranna", distance: "0.71 km", activities: ["kibel", "karton", "kuchnia", "ryj człowieka"]}
+]
+
+
+
+export function TripsListElement() {
+  return (
+    <>
+      <PostsList Content={trips.map((trip: TripData) => (<TripPost data={trip} width={600}/>))}/>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)